### DISCUSSION TITLE
Rewrites to host multiple subdomains under one domain

### DISCUSSION THREAD

**Discussion by ptrhck** [vercel/community#137] [Help] [answered]
Following my approach https://github.com/facebook/docusaurus/discussions/6086 to host multiple docusaurus sites under one domain, I am trying to add the correct `rewrite `config to my `vercel.json`.

Assuming the main repo is hosted under `docs.example.com`. Now, I would like to rewrite the standalone docusaurus deployments `ios.docs.example.com` to `docs.example.com/ios` and `android.docs.example`.com to `docs.example.com/android`.

The idea here is that visiting `docs.example.com/ios` will display the content of `ios.docs.example.com`, e.g.
`ios.docs.example.com/docs/intro` -> `docs.example.com/ios/docs/intro`

In which project would I need to add a rewrite config and what should it look like?

**Comment by Pieparker** [comment, marked as answer]
> In which project would I need to add a rewrite config

The one which `docs.example.com/ios` is attached to.

>what should it look like?

Something like:
```
{
  "rewrites": [
    { "source": "/ios*", "destination": "https://ios.docs.example.com*" },
  ]
}
```
But you may need or want to tweak this to your liking.
